setup.py

```diff
@@ -118,7 +118,8 @@ def package_version():
             'pytorch.js', 'pytorch-metadata.json', 'pickle.js',
             'sklearn.js', 'sklearn-metadata.json',
             'tf.js', 'tf-metadata.json', 'tf-proto.js', 
-            'tflite.js', 'tflite-metadata.json', 'tflite-schema.js', 
+            'tflite.js', 'tflite-metadata.json', 'tflite-schema.js',
+            'openvino-ir.js', 'openvino-ir-parser.js',
```

Comment:
`openvino.js`, `openvino-parser.js`

Reply:
renamed

is the discussion resolved now?